.env

```diff
@@ -32,5 +32,8 @@ MYSQL_USER=root
 MYSQL_PASSWORD='12345'
 MYSQL_DB=devopsloft
 MYSQL_ROOT_HOST="%"
+
+VAULT_GUEST_PORT=8200
```

Comment:
Meaningless if it's not used in the hcl to listen on the port

Reply:
It is used in the Vagrantfile. How do you suggest it should be used?

Reply:
It's used only in the Vagrantfile, but if the vault will not actually listen to this port (see `vault_s2i/config.hcl:4` and `vault_s2i/config.hcl:12` then it's meaningless and you will have broken environment

Reply:
agree. Will make config.hcl a jinja2 template file.